packages/admin/dashboard/src/extensions/dashboard-extension-manager/dashboard-extension-manager.tsx

```diff
@@ -117,6 +117,7 @@ export class DashboardExtensionManager {
         to: item.path,
         icon: item.icon ? <item.icon /> : undefined,
         items: [],
+        nested: item.nested,
```

Comment:
Here it would be good to log a warning if the item has a `nested` field 
but also has children since they will be ignored.

If you create a nested route for `/brand` like this:

```
// /src/admin/routes/brands/page.tsx
import { defineRouteConfig } from "@medusajs/admin-sdk";

const BrandsPage = () => {
  return <div>Brands</div>;
};

export const config = defineRouteConfig({
    label: "Brands",
    nested: "/products",
})

export default BrandsPage;

// src/admin/routes/brands/create/route.tsx
import { defineRouteConfig } from "@medusajs/admin-sdk";

const CreateBrandPage = () => {
  return <div>Create Brand Page</div>;
};

export const config = defineRouteConfig({
    label: "Create Brand", // This would normally generate a nested menu item under "Brands".
})

export default CreateBrandPage;
```

The nested MenuItem is created, but never used.

So when looping over the items, it would be great to explicitly warn the user why this item is not being used. We need to keep track of items that have a `nested` field, if we later come across an item which is a child of the item with a `nested` we should skip it, and warn that we are doing so. A little further up in the function you can see we do something similar for other cases.

Reply:
Good point. But I find out some challange here. The new parameter `nested` passing only on the first route. 
That is means that we cannot check parameter `nested` on the nested page `create`. I hope I described it clear)

Possible solution: locate a new route under existing route by path:
```
// /src/admin/routes/products/brands/page.tsx - located under products route
import { defineRouteConfig } from "@medusajs/admin-sdk";

const BrandsPage = () => {
  return <div>Brands</div>;
};

export const config = defineRouteConfig({
    label: "Brands",
})

export default BrandsPage;

// /src/admin/routes/products/brands/create/page.tsx - will be ignored
import { defineRouteConfig } from "@medusajs/admin-sdk";

const CreateBrandPage = () => {
  return <div>Create Brand Page</div>;
};

export const config = defineRouteConfig({
    label: "Create Brand",
})

export default CreateBrandPage;
```

In this case `nested` param will be deleted. 

What do you think about that? Or maybe you have a better solution for that?

Reply:
Good point. But I find out some challange here. The new parameter `nested` passing only on the first route. 
That is means that we cannot check parameter `nested` on the nested page `create`. I hope I described it clear)

Possible solution: locate a new route under existing route by path:
```
// /src/admin/routes/products/brands/page.tsx - located under products route
import { defineRouteConfig } from "@medusajs/admin-sdk";

const BrandsPage = () => {
  return <div>Brands</div>;
};

export const config = defineRouteConfig({
    label: "Brands",
})

export default BrandsPage;

// /src/admin/routes/products/brands/create/page.tsx - will be ignored
import { defineRouteConfig } from "@medusajs/admin-sdk";

const CreateBrandPage = () => {
  return <div>Create Brand Page</div>;
};

export const config = defineRouteConfig({
    label: "Create Brand",
})

export default CreateBrandPage;
```

In this case `nested` param will be deleted. 

What do you think about that? Or maybe you have a better solution for that?

Reply:
I found a more easy solution. We have `menuItems` and we can find nested route there. :)

Reply:
Added warning